src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -194,8 +194,25 @@ void ov::npuw::util::unpack(const ov::SoPtr<ov::ITensor>& from,
             NPUW_ASSERT(false);
         }
     } else if (type_from == ov::element::u8) {
-        // Only support CW for now
-        if (scale_shape.size() == 2 && scale_shape[0] == from_shape[0] && scale_shape[1] == 1) {
+        if (scale_shape.size() == 3 && scale_shape[1] == 1 && scale_shape[2] == 1) {
+            // Special case for broadcasting vocab by 2 dimensions
+            // FIXME: all this logic probably should be in some specific unpack or another util function
+            ov::Tensor wraped_from(from->get_element_type(), ov::Shape{from_shape[0], from_shape[1] * from_shape[2]});
+            const auto& zerop_shape = zerop->get_shape();
+            ov::Tensor wraped_zerop(zerop->get_element_type(),
+                                    ov::Shape{zerop_shape[0], zerop_shape[1] * zerop_shape[2]});
+            ov::Tensor wraped_scale(scale->get_element_type(),
+                                    ov::Shape{scale_shape[0], scale_shape[1] * scale_shape[2]});
+            ov::npuw::util::merge_last_two_dims(from, ov::get_tensor_impl(wraped_from));
+            ov::npuw::util::merge_last_two_dims(zerop, ov::get_tensor_impl(wraped_zerop));
+            ov::npuw::util::merge_last_two_dims(scale, ov::get_tensor_impl(wraped_scale));
```

Comment:
```suggestion
            ov::Tensor wraped_zerop(zerop->get_element_type(),
                                    ov::Shape{zerop_shape[0], zerop_shape[1] * zerop_shape[2]},
                                    zerop->data(), zerop->get_strides());
            ov::Tensor wraped_scale(scale->get_element_type(),
                                    ov::Shape{scale_shape[0], scale_shape[1] * scale_shape[2]},
                                    scale->data(), scale->get_strides());
```

Reply:
same for weights

Reply:
Dropped the strides though, not sure how to calculate them

Reply:
you don't need to calculate those, you just need pass one from another as I've shared in my example.